tools/development/parser/types.c

```diff
@@ -28,20 +28,77 @@ gst2pbtxt_parse_error_quark (void)
   return quark;
 }
 
+/** @brief Internal function to construct _Element */
+static void
+_nnstparser_config_element (_Element * e, const gchar * name)
+{
+  e->specialType = eST_normal;
+  e->element = g_strdup (element);
```

Comment:
error, element is arg?

Reply:
It was missing. Thanks for pointing it out. It will be fixed in the next iteration.